pyproject.toml

```diff
@@ -28,7 +28,7 @@ test = [
 docs = [
     "Sphinx==5.1.1",
     "sphinx-autodoc-typehints==1.19.1",
-    "ipython==8.11.0",
+    "ipython==7.34.0",
```

Comment:
Dependencies should be updated.

Reply:
lets `pip-compile` pin versions for `optional-dependencies`.